apis/project.cattle.io/v3/app_types.go

```diff
@@ -29,6 +29,7 @@ type AppSpec struct {
 	Prune               bool              `json:"prune,omitempty"`
 	MultiClusterAppName string            `json:"multiClusterAppName,omitempty" norman:"type=reference[/v3/schemas/multiclusterapp]"`
 	ValuesYaml          string            `json:"valuesYaml,omitempty"`
+	HelmVersion         string            `json:"helmVersion,omitempty" norman:"nocreate,noupdate"`
```

Comment:
Let's move this to status instead. Spec indicates something that a user sets while status is controlled internally. Also in the future status will be protected similar to how k8s does it but through 'steve'.